app/models/energy/ScheduleFixedInterval.py

```diff
@@ -0,0 +1,63 @@
+"""Schedule Fixed Interval Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError, UrlStr, ConstrainedStr
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+import datetime
+from app.models.energy.ScheduleBase import ScheduleContinuous, ScheduleDiscrete, ScheduleNumericType, ScheduleUnitType, Date, Time
```

Comment:
It looks like you are not using most of these objects. You should only import what is going to be used.